src/test/utils.ts

```diff
@@ -1 +1,18 @@
+import { db } from '../database';
+import type { Result } from 'tinypg';
+
 export const isoTimestampPattern = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{3}Z$/;
+
+interface TableMetrics {
+  count: number;
+  maxId: number;
+  now: Date;
+}
+export const getTableMetrics = async (tableWithId: string): Promise<Result<TableMetrics>> => (
```

Comment:
Might as well return the metrics rather than the result since I don't think the result is ever going to be interesting (famous last words maybe)